src/main/java/seedu/address/logic/commands/DisinviteCommand.java

```diff
@@ -135,8 +136,10 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         // TODO: check disinvite function again with the updated expense
+        Context newContext = new Context(activityToDisinviteFrom);
+        model.setContext(newContext);
 
-        return new CommandResult(result);
+        return new CommandResult(result, newContext);
```

Comment:
Same as for the invite command, we should move this into the success `else` block and prompt the re-render only when necessary.